src/bin/dual-iir.rs

```diff
@@ -62,13 +62,17 @@ const APP: () = {
 
         let mqtt_interface = {
             let mqtt_client = {
-                let broker = IpAddr::V4(Ipv4Addr::new(10, 34, 16, 1));
+                let broker = IpAddr::V4(Ipv4Addr::new(10, 34, 16, 10));
```

Comment:
Lets pull this default broker address out to a single config file so it stays consistent for all of the binaries. Maybe `design_parameters.rs`?

Reply:
Ack